Assets/Vimeo/Scripts/Config/VimeoVideo.cs

```diff
@@ -133,7 +154,25 @@ public override string ToString()
         {   
             return name;
         }  
-        
+
+        public string getDashUrl()
+        {
+            if (dashUrl != null) {
+                return dashUrl;
+            }
+            else if (getHlsUrl() != null) {
+                Debug.LogWarning("[Vimeo] No DASH manfiest found. Defaulting to HLS.");
+                return getHlsUrl();
+            }
+
+            return null;
+        }
+
+        public string getHlsUrl()
+        {
+            return hlsUrl;
```

Comment:
Should we not test that `hlsUrl != null` and if it is null `Debug.LogWarning` the same way we do in `getDashUrl`?

Reply:
I only did this for Dash because Dash is not returned in a custom vimeo dev app. But actually, probably not good practice to do that in these methods... if you call `getDashUrl` you probably expect it to return Dash or `null`... and not an HLS fallback